### PR TITLE
Filter for video notes in the command /lock

### PR DESCRIPTION
**Pull request by IGUNUBLUE**
The video notes when applying the command lock all, were not filtered, You can also filter individually as with the other locktypes. I include the changes in the respective sql.

### CODE REVIEW

File: tg_bot/modules/sql/locks_sql.py [hunk 1]
```diff
@@ -14,6 +15,7 @@ class Permissions(BASE):
     voice = Column(Boolean, default=False)
     contact = Column(Boolean, default=False)
     video = Column(Boolean, default=False)
+    videonote = Column(Boolean, default=False)
```
Comment (PaulSonOfLars) [inline tg_bot/modules/sql/locks_sql.py]: What is the sql command to add this new column, for the migration to complete successfully? Sqlalchemy does not automatically migrate.

File: tg_bot/modules/sql/locks_sql.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 from sqlalchemy import Column, String, Boolean
 
 from tg_bot.modules.sql import SESSION, BASE
+from telegram.files import videonote
```
[the remark this answers is not in the document]
Comment (IGUNUBLUE) [reply tg_bot/modules/sql/locks_sql.py]: I added my IDE, sorry.

File: tg_bot/modules/locks.py [hunk 3]
```diff
@@ -249,6 +250,7 @@ def build_lock_message(chat_id):
                    "\n - voice = `{}`" \
                    "\n - document = `{}`" \
                    "\n - video = `{}`" \
+                   "\n - videonote = `{}`" \
```
[the remark this answers is not in the document]
Comment (IGUNUBLUE) [reply tg_bot/modules/locks.py]: I just corrected it

File: .gitignore [hunk 4]
```diff
@@ -1,3 +1,7 @@
 tg_bot/config.py
 *.pyc
 .idea/
+.project
+.pydevproject
+.directory
+.vscode
```
[the remark this answers is not in the document]
Comment (IGUNUBLUE) [reply .gitignore]: are files that my IDE creates, eclipse-pydev

File: tg_bot/modules/sql/locks_sql.py (hunk 1 above)
Comment (IGUNUBLUE) [reply tg_bot/modules/sql/locks_sql.py]: ups! You are right, can you give me an idea of how to do it?

**Comment by IGUNUBLUE** [comment]
I've tried it and it works, I must say that my experience is not like yours, the most sure is that I have mistakes but is not that the way to learn? I honestly do it from the heart, I just want to help. please, do not get angry with me.

**Pull request merged**